fixit/cli/main.py

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Usage:
+#
+#   $ fixit run_rules --help
+#   $ fixit apply_fix --help
+#   $ fixit insert_suppressions --help
+#   $ fixit add_new_rule --help
+
+import argparse
+import importlib.util
+from importlib.machinery import ModuleSpec
+from pathlib import Path
+from types import ModuleType
+from typing import List
+
+from absl import app
+from absl.flags import argparse_flags
```

Comment:
Why do we need `absl`?
Doesn't the builtin `argparse` already provide the same features (`ArgumentParser`  and `add_subparsers`)?

Reply:
I have created CLI using `absl` in the past. So, implemented this way.
Will remove `absl` module and use only `argparse`.

Reply:
removed `absl` support